packages/edit-site/src/components/add-new-template/index.js

```diff
@@ -107,8 +107,7 @@ function TemplateListItem( {
 } ) {
 	return (
 		<Button
-			// TODO: Switch to `true` (40px size) if possible
-			__next40pxDefaultSize={ false }
+			__next40pxDefaultSize
```

Comment:
This can be seen when you go to `site editor` and go to template, add new and check the modal for the same.

**Note:** This would have no impact as it has some styles which are overridden.

| Before | After     |
| :-------- | :------- |
| <img width="1369" alt="custom-template-index-before" src="https://github.com/user-attachments/assets/97185792-23f2-4785-a48a-9cb508c1f704"> | <img width="1369" alt="Custom-template-index-after" src="https://github.com/user-attachments/assets/31c4ba57-dfc2-4d90-b2c8-dd543c83f8a1"> |